internal/testrunner/runners/static/test_config.go

```diff
@@ -0,0 +1,47 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package static
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/elastic/go-ucfg"
+	"github.com/elastic/go-ucfg/yaml"
+	"github.com/pkg/errors"
+
+	"github.com/elastic/elastic-package/internal/testrunner"
+)
+
+type testConfig struct {
+	testrunner.SkippableConfig `config:",inline"`
+}
+
+func newConfig(staticTestFolderPath string) (*testConfig, error) {
+	configFilePath := filepath.Join(staticTestFolderPath, "config.yml")
```

Comment:
Should we define this as part of the spec, as an optional file?

Reply:
You're right, I totally forgot about the package-spec.

BTW Should I wait with this PR until you'll merge https://github.com/elastic/elastic-package/pull/260 or do you prefer the other way round?

Reply:
https://github.com/elastic/package-spec/pull/141

Reply:
I just merged #260 so you'll need to rebase this PR on top of `master`.